slices/sensible-utils.yaml

```diff
@@ -0,0 +1,20 @@
+package: sensible-utils
+
+slices:
+  base:
+    essential:
+      - sensible-utils_bins
+      - sensible-utils_data
```

Comment:
It seems to me that we are using the `base` slices (in some other places too) to contain pretty much the standard almost-everything bits from a package. The problem here would be if we start listing the `base` slices as dependencies instead of something small which would meet the necessities, for example, `bins`. In that case our rootfs will contain stuff we do not really need.
Let me know what you think about this.

Reply:
Updated this based on our discussion